### ISSUE TITLE
Ignore non-passenger and private ferries

### ISSUE THREAD

**Issue by ZeLonewolf** [openmaptiles/openmaptiles#1673]
Right now, we include ferries which are not typically rendered in general-purpose maps.  The issue ZeLonewolf/openstreetmap-americana#1130 shows an example where a private freight "ferry" line which crosses the Gulf of Mexico is rendered at low zoom. I propose that we ignore ferry lines tagged `access=private` and potentially even `railway:traffic_mode=freight`.

**Comment by quincylvania** [comment]
Agreed that private/freight ferries should be ignored. There was a similar case where the freight ferry from Washington to Alaska was mapped in OSM, but I'm not seeing it now. Proper tagging and rendering can remove the impulse to delete the data.

**Comment by talllguy** [comment]
Thanks for considering this and for your comments @ZeLonewolf on the Americana repo. I figured it might be a ferry or maybe one of those pesky time zone lines or maybe even the EEZ. I appreciate the analysis!

I do agree that a public ferry of regional import would be good to show on a general purpose map. The Cape May Lewes ferry is a good example. It is open to the public and signposted all over the Delmarva Peninsula. This train ferry on the other hand is more appropriate for a map of commerce.
